recipes/zmqpp/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans import AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+class ZmqppConan(ConanFile):
+    name = "zmqpp"
+    homepage = "https://github.com/zeromq/zmqpp"
+    version = "4.2.0"
+    version_major = 4
+    license = "MPLv2"
```

Comment:
We use the SDPX to stay official https://spdx.org/licenses/MPL-2.0.html

```suggestion
    license = "MPL-2.0"
```